docs/RFCS/00000000_template.md

```diff
@@ -75,6 +81,10 @@ Why should we *not* do this?
 
 If applicable, list mitigating factors that may make each drawback acceptable.
 
+If drawbacks include potential impacts on other areas of the product, identify tests
+to run before marking the feature as complete to make sure we don't introduce incomplete
```

Comment:
This may be worth talking in person; I don't think running tests solves the problem here. This is getting out and actually talking to folks too. Potentially even comms plans if this changes customer behavior. Materially changing other areas of the product is a big deal.

Reply:
Yeah the phrasing is also dangerous, because if a feature does not have natural "tests" this requirement can completely stall progress on the RFC.

I would suggest instead:

"
Investigate the consequences of the proposed change onto other areas of CockroachDB. If other features are impacted, especially UX, list this impact as a reason not to do the change. If possible, also investigate and suggest mitigating actions that would reduce the impact. You can for example consider additional validation testing, additional documentation or doc changes, new user research, etc.
"

Reply:
Also I just noticed your initial remark about performance regression. This needs a separate paragraph altogether:

"
Also investigate the consequences of the proposed change on performance. Pay especially attention to the risk that introducing a possible performance improvement in one area can slow down another area in unexpected way. Examine all the current "consumers" of the code path you are proposing to change and consider whether any of them may be negatively impacted by the proposed change. List all these consequences as possible drawbacks.
"